app/View/Components/Photo.php

```diff
@@ -27,8 +27,8 @@ class Photo extends Component
 	public $srcset2x = '';
 
 	public $layout = false;
-	public $_w = 200;
-	public $_h = 200;
+	public int $_w = \App\Models\Photo::THUMBNAIL_DIM;
```

Comment:
you can do
```php
use App\Models\Photo as PhotoModel;
```
on the top of the file. This will alias the path (damn I really miss this when I do Java).

Reply:
👍 I'll incorporate that into the update of this PR.

Reply:
Fixed in commit c513d92d.